.setup/pip/system_requirements.txt

```diff
@@ -24,7 +24,7 @@ psutil==5.9.8
 python-dateutil==2.8.2
 watchdog==4.0.1
 # Held back from 0.8.2 as no whl is available which breaks installing on Ubuntu 22.04
-xlsx2csv==0.8.1
+xlsx2csv==0.8.3
```

Comment:
@powe97 The comment above is no longer relevant.  Can you please remove it?

Reply:
Sure, I missed that.